recipes/boto3/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "boto3" %}
+{% set version = "1.3.1" %}
```

Comment:
Should we bump to 1.4.0 as that is [out]( https://pypi.python.org/pypi/boto3/1.4.0 )?